src/unix/linux_like/linux/gnu/b32/mod.rs

```diff
@@ -204,23 +204,74 @@ s! {
 
     pub struct timex {
```

Comment:
Maybe this should be split in two to avoid interleaving fields?

Reply:
Done